catalog/kafkamanager/1.3.3/dockerfile/Dockerfile

```diff
@@ -18,7 +18,10 @@ RUN set -ex; \
   cd /tmp/kafka-manager; \
   git checkout ${KM_REVISION}; \
   echo 'scalacOptions ++= Seq("-Xmax-classfile-name", "200")' >> build.sbt; \
-  ./sbt clean dist; \
+  ./sbt clean dist 2>&1 | tee sbt-out.log; \
+  while $(grep -q FAILED sbt-out.log); do \
```

Comment:
The dead loop does not look good. The sbt download issue would be a temporary issue. Could simply retry the build.

Reply:
Yeah, I agree. I can add a counter to limit attempts to, say, 10. Will this work for you?